src/kernel/arch/aarch64/tty.zig

```diff
@@ -2,12 +2,13 @@ const std = @import("std");
 const arch = @import("arch.zig");
 const TTY = @import("../../tty.zig").TTY;
 const panic = @import("../../panic.zig").panic;
-const log = @import("../../log.zig");
+const log = @import("../../log.zig").log;
```

Comment:
We've transitioned to the logger in the standard library, so this can be changed to `const log = std.log.scoped(.aarch64_tty);`

Reply:
Done